content/en/tutorials/pixie-101/request-tracing.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Request Tracing"
+metaTitle: "Tutorials | Pixie 101 | Request Tracing"
+metaDescription: "Learn how to use Pixie to trace requests in your cluster."
+order: 5
+---
+
+The move from monolith to microservice architecture has greatly increased the volume of inter-service traffic. Pixie makes debugging  this communication between services easy by providing immediate and deep (full-body) visibility into requests flowing through your cluster.
+
+HTTP requests are featured in this tutorial, but Pixie can trace a number of different protocols including DNS, PostgreSQL, and MySQL. See the full list [here](/about-pixie/data-sources/#supported-protocols).
+
+This tutorial will demonstrate how to use Pixie to:
+
+- Inspect full-body HTTP requests.
+- See HTTP error rate per service.
+- See HTTP error rate per pod.
+
+If you're interested in troubleshooting HTTP latency, check out the [Service Performance](/tutorials/pixie-101/service-performance) tutorial.
+
+## Prerequisites
+
+1. You will need a Kubernetes cluster with Pixie installed. If you do not have a cluster, you can create a minikube cluster and install Pixie using our [installation steps](/installing-pixie/).
+
+2. You will need an application that makes HTTP requests. To install a demo app that uses HTTP:
+
+> - [Install the Pixie CLI](/installing-pixie/install-schemes/cli/#1.-install-the-pixie-cli)
+> - Run `px demo deploy px-sock-shop` to install Weavework's [Sock Shop](https://microservices-demo.github.io/) demo app.
+> - Run `kubectl get pods -n px-sock-shop` to make sure all pods are ready before proceeding. The demo app can take up to 5 minutes to deploy.
+
+## Full-Body HTTP Request
+
+A developer has noticed that the demo application's `cart` service is reporting errors.
```

Comment:
I feel like this tutorial just jumped into this scenario very suddenly. I would give some quick background like "In this tutorial we will be following a scenario where... "

Or in the prereqs, it just says "You will need an application that makes HTTP requests. To install a demo app that uses HTTP:"... Maybe we should say "To follow along to this tutorial, install the demo app"